cobbler/settings/migrations/V3_3_0.py

```diff
@@ -38,7 +38,7 @@
         'i686': {'binary_name': 'bootia32.efi'},
         'IA64': {'binary_name': 'bootia64.efi'},
         'powerpc-ieee1275': {'binary_name': 'grub.ppc64le', 'extra_modules': ['net', 'ofnet']},
-        'x86_64-efi': {'binary_name': 'grubx86.efi', 'extra_modules': ['chain', 'efinet']}}
+        'x86_64-efi': {'binary_name': 'grubx64.efi', 'extra_modules': ['chain', 'efinet']}}
```

Comment:
Please don't touch this file! Please add these changes into a new migration. Old migrations should never change!